README.md

```diff
@@ -843,7 +843,7 @@ render status: :forbidden
     User.find_by_first_name_and_last_name('Bruce', 'Wayne')
 
     # good
-    User.find(id)
+    User.find_by(email: email)
```

Comment:
The 2 corresponding `#bad` examples above should also use `email` instead of `id`.